src/pudl/transform/eia860.py

```diff
@@ -1250,10 +1214,8 @@ def _core_eia860__cooling_equipment(
     ce_df.loc[:, ce_df.columns.str.endswith("_thousand_dollars")] *= 1000
     ce_df.columns = ce_df.columns.str.replace("_thousand_dollars", "")
 
-    resource = pudl.metadata.classes.Package.from_resource_ids().get_resource(
-        "_core_eia860__cooling_equipment"
-    )
-    return ce_df.pipe(apply_pudl_dtypes, group="eia", strict=True).pipe(resource.encode)
+    # Encoding is required here because this table is not yet getting harvested.
+    return apply_pudl_dtypes(ce_df, group="eia", strict=True).pipe(PUDL_PACKAGE.encode)
```

Comment:
Left the encoding step here because this table does not currently get passed into the harvesting process.

**Question:** Based on the name, this table shouldn't be getting written to the database, but it is, because it's the only version of this data that's available right now.  Is that how we want to do this? It's a little confusing.

Reply:
But it's not harvested and normalized like the other `core_` tables, so that also seems a little bit off.